### ISSUE TITLE
Documentation link is redirected

### ISSUE THREAD

**Issue by jonathanunderwood** [openwisp/netjsonconfig#143]
Clicking on the documentation link in the README briefly loads the documentation, and then the browser is redirected to:

http://ww38.nodeshot.org/github-btn.html?user=openwisp&repo=netjsonconfig&type=watch&count=true&size=large&_xafvr=MWI1MGI3MmEwMGRlNWQ2YjMyNzgyYTZjN2EwYzYzNDdmOWM2ZTJhOCw1ZTAzMmMzMTIyZTdl

**Comment by IAmAMissingColon** [comment]
+1 Same here ! But I thought it was browser acting weird

**Comment by atb00ker** [comment]
Thanks for report, this seems to be unintensional and some kind of misconfiguration.
Might be related to: https://github.com/openwisp/netjsonconfig/blob/ec56134759866919040db0015c3acb77180a5091/docs/source/_github.rst

But we need to investigate further to solve it.

**Comment by IAmAMissingColon** [comment]
is this issue fixed ?

**Comment by jonathanunderwood** [comment]
Yes, it appears to be fine now.

**Comment by jonathanunderwood** [comment]
The question is: how did this happen in the first place? Subversively collecting geolocation data is not what I expect from an open source project.

**Comment by nemesisdesign** [comment]
I updated the contents of the docs with the updated link: https://github.com/openwisp/netjsonconfig/commit/ddc63c3e8e0a6742baca4a7e188e46f89603e645.
But we need to issue a new release in order to have it updated on the generated docs.
The older docs may stay like that forever unless we delete them.

> The question is: how did this happen in the first place? Subversively collecting geolocation data is not what I expect from an open source project.

What are you talking about? I was the maintainer of another project which the new version of OpenWISP initially based some code on, which was called [nodeshot](https://github.com/ninuxorg/nodeshot), and I used the domain I had to serve some static content which displayed a star and watch on github button, then the domain expired.

You've been pretty fast to jump to negative conclusions.
